### ISSUE TITLE
Reddit redirect links adding https

### ISSUE THREAD

**Issue by monogoat** [burhan-syed/troddit#103]
For a while I thought my problem was the add-on I was using to redirect reddit links to my local troddit instance, but looking into it today I think I've narrowed the issue down to troddit's internal link rewriting.

My local instance is not using SSL, so it is accessed via http://. When clicking on a reddit link from within troddit, the links are rewritten from https://www.reddit.com to https://IP:PORT which then fails to open with an SSL related error. I need to delete the s from after http in the address bar to continue to the post.

Perhaps the rewrite is only targeting after the slashes, but could be changed to include the http portion as well?

**Comment by prime-optimal** [comment]
i'm in the same boat.  just gave myself a crash course in generating certificates and keys so I could secure my Portainer instance hoping I could do the same for troddit.  🧐

**Comment by monogoat** [comment]
I'm sorry for the delayed response, I stopped using a local instance for a while after noting this issue, but now that reddit is being dumb I created a new instance on a vps. Nginx and firewalls fought with me and I gave up messing with it and am just connecting via IP:PORT without SSL again, and this issue seems to persist despite the code changes you made. I can send you the address of my instance if you'd like it to test.
